post_installation_configuration/cluster-tasks.adoc

```diff
@@ -0,0 +1,83 @@
+:context: post-install-cluster-tasks
+[id="post-install-cluster-tasks"]
+= Post-installation cluster tasks
+include::modules/common-attributes.adoc[]
+toc::[]
+
+[%interactive]
+* [ ] If you incorrectly sized the worker nodes during deployment, adjust them by
+creating one or more new MachineSets, scale them up, then scale the original
+MachineSet down before removing them.
+* [ ] If desired, create and configure additional MachineSets to dedicate for specific
+workloads.
+* [ ] Decide whether to enable or disable specific feature gates.
+* [ ] When using a full-stack automation capable platform, enable and configure cluster autoscaling.
+* [ ] Review recommended etcd practices for large and dense clusters.
+* [ ] Configure etcd encryption.
+* [ ] Back up etcd.
+* [ ] Test cluster restore functionality.
+* [ ] Configure the Pod disruption budget to prevent accidental outages.
+
+[id="post-install-adjust-worker-nodes"]
+== Adjust worker nodes
+
+If you incorrectly sized the worker nodes during deployment, adjust them by
+creating one or more new MachineSets, scale them up, then scale the original
+MachineSet down before removing them.
+
+include::modules/machineset-manually-scaling.adoc[leveloffset=+2]
+
+[id="post-install-creating-infrastructure-machinesets"]
+== Creating infrastructure MachineSets
+
+include::modules/infrastructure-components.adoc[leveloffset=+2]
+
+[id="post-install-creating-infrastructure-machinesets-production"]
+== Creating infrastructure MachineSets for production environments
+
+In a production deployment, deploy at least three MachineSets to hold
+infrastructure components. Both the logging aggregation solution and
```

Comment:
This should be split up and put on the operator pages for each component. Not all customers install the logging stack, not all customers install Service Mesh, as such if you start with the first sentence; its not clear why exactly I need 'at least' three MachineSets. 

IMO this should be re-worked to tell readers that they need to setup MachineSets and pin workloads to them as they see fit. Then ... Each Operator / product can explain how there deployment/install can work with the structures OCP and MachineSets provides.